src/test/main.ml

```diff
@@ -581,14 +581,20 @@ let () = add_tests @@ List.concat [
       );
   ]
 
-let () = add_tests @@ exits 25 Construct.(
+let () = add_tests @@ exits ~name:"getenv" 25 Construct.(
+    let alternate_get_env v =
+      (* We cannot use OCaml's Sys.getenv because the compilation output may
+         be run on a different host/system (through SSH or alike). *)
+      exec ["sh"; "-c";
+            sprintf "echo ${%s} | tr -d '\\n'" v
+           ] |> output_as_string in
```

Comment:
This seems a bit weird, without looking at the implementation of `getenv`, how would I know that this is different?